### PR TITLE
iboview: init at 20211019

### PR DESCRIPTION
**Pull request by sheepforce**

### CODE REVIEW

File: pkgs/apps/iboview/default.nix [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{ lib, stdenv, wrapQtAppsHook, qmake, qtscript, qtsvg, blas-ilp64, lapack-ilp64
+, glew, boost, eigen, libGLU }:
+
+stdenv.mkDerivation rec {
+  pname = "IboView";
+  version = "20211019";
+
+  src = builtins.fetchurl {
```
Comment (markuskowa) [inline pkgs/apps/iboview/default.nix:8]:
```suggestion
  src = fetchurl {
```

That should be the `fetchurl` from nixpkgs or I get:
```
error: access to URI 'http://www.iboview.org/bin/ibo-view.20211019.tar.bz2' is forbidden in restricted mode
```

File: pkgs/apps/iboview/default.nix [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  # Standard QMake installation fails.
+  installPhase = ''
+    mkdir -p $out/bin
+    cp iboview $out/bin/.
+  '';
+
+  meta = with lib; {
+    description = "Calculator and visualiser for Intrinsic Bond Orbitals";
+    homepage = "http://www.iboview.org/index.html";
+    license = licenses.gpl3Plus; # Although the README has some quite interesting statements on copyright
```
Comment (markuskowa) [inline pkgs/apps/iboview/default.nix:43]: >  Although the README has some quite interesting statements on copyright

Looks fine to me.

**Comment by markuskowa** [comment]
```
hash mismatch in fixed-output derivation '/nix/store/83m6368yxyknfgzx8srgrpxdmdwmjjaa-ibo-view.20211019.tar.bz2':
  wanted: sha256:0ihd09cicya9zz77j259a9kvj0jsrczsi21dalldp1zxg7hzj17b
  got:    sha256:0mrspsdl30n2v5ymrig7yqq6xmpad9r3zr0da5hp7b8gzyzkg61f
```

**Comment by markuskowa** [comment]
I think the hash mismatch comes from switching the fetcher.

**Comment by sheepforce** [comment]
Hm, no if i change to the new hash, `fetchurl` complains and indeed tells me the old one is correct. :thinking:

**Comment by sheepforce** [comment]
`nix-prefetch-url` then again agrees with your new hash :sweat_smile:

**Comment by markuskowa** [comment]
I can build it manually with `0mrspsdl30n2v5ymrig7yqq6xmpad9r3zr0da5hp7b8gzyzkg61f`

**Comment by sheepforce** [comment]
> I can build it manually with `0mrspsdl30n2v5ymrig7yqq6xmpad9r3zr0da5hp7b8gzyzkg61f`

The current version does not work for you? It works on my workstation and laptop.

**Comment by markuskowa** [comment]
> > I can build it manually with `0mrspsdl30n2v5ymrig7yqq6xmpad9r3zr0da5hp7b8gzyzkg61f`
> 
> The current version does not work for you? It works on my workstation and laptop.

No, it does not work for me. If you still have the old version in your nix store, nix will not make another download attempt and just use the one with the old hash.

**Comment by sheepforce** [comment]
Alright, here is the updated version with your hash.

> If you still have the old version in your nix store, nix will not make another download attempt

I know, but I thought it was a fresh build on my laptop. Apparently not :)

**Review by markuskowa — APPROVED** [review]

**Pull request merged**